CITATION.cff

```diff
@@ -0,0 +1,19 @@
+cff-version: 1.2.0
+message: "If you've used tephi in your research, please cite us using the references below."
+authors:
+- family-names: "Little"
+  given-names: "Bill"
+  orcid: "https://orcid.org/0000-0002-1345-9465"
+- family-names: "Sadek"
+  given-names: "Elias"
+  orcid: "https://orcid.org/0009-0007-3284-9745"
+
+title: "tephi"
+abstract: "Tephigram plotting in Python"
+version: "0.2.4"
```

Comment:
@ESadek-MO We don't include the version here, otherwise we'll have to constantly update it.

Reply:
<3